virtual-server/examples/go/main.go

```diff
@@ -190,44 +197,57 @@ runcmd:
 	// Add the example PVC as a file system to the Virtual Server
 	virtualServer.AddPVCFileSystem("example-storage", pvc.Name, false)
 
-	service := buildFloatingIPService("example-floating-ip-service", namespace)
-	if err := c.Create(context.Background(), service); err != nil {
-		log.Fatalf("Could not create example floatingIP service\nReason: %s", err.Error())
-	}
+	service := &corev1.Service{}
+	if floatingServiceExist {
+		log.Printf("Creating floating service %s", floatingServiceName)
+		service = buildFloatingIPService(floatingServiceName, namespace)
+		if err := c.Create(context.Background(), service); err != nil {
+			log.Fatalf("Could not create example floatingIP service, reason: %s", err.Error())
+		}
 
-	// Add the example floatingIP service to the VirtualServer
-	virtualServer.AddFloatingIP(service.Name)
+		// Add the example floatingIP service to the VirtualServer
+		virtualServer.AddFloatingIP(service.Name)
+	} else {
+		log.Print("Floating service disabled")
+	}
```

Comment:
Probably don't need this else block. Not really useful info if they didn't specify

Reply:
removed